docs/DeveloperGuide.md

```diff
@@ -459,10 +459,12 @@ The income or spending will be deleted from its corresponding list using its ind
 
 ## Product scope
 ### Target user profile
-1. prefer desktop apps over other types
+1. Prefer desktop apps over other types
```

Comment:
other types seems a bit too general, maybe leave it out or be more specific?

Reply:
Fixed!